rest/authorization-server/src/main/resources/application.yml

```diff
@@ -61,13 +64,13 @@ security:
   accesstoken:
     validity: 30
 
-#logging:
-#  level:
-#    org.springframework.security: DEBUG
-#    org.springframework.security.oauth2: DEBUG
-#    org.springframework.security.oauth2.provider.endpoint: DEBUG
-#    org.springframework.security.oauth2.provider.error: DEBUG
-#    org.springframework.security.oauth2.provider.token: DEBUG
-#    org.springframework.security.oauth2.provider.token.store: DEBUG
-#    org.springframework.security.oauth2.provider.token.store.jwk: DEBUG
+logging:
```

Comment:
Should we always output logs with "DEBUG" level?

Reply:
No, but we are in the deep development transition. I think is advisable to us now keep the REST in debug mode before release, as it became the most important part of our entire system

Reply:
OK we set debug log level temporary.